docs/monitors/statsd.md

```diff
@@ -0,0 +1,110 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# statsd
+
+
+
+### USAGE
+
+#### Adding dimensions to StatsD metrics
+
+The StatsD monitor can parse keywords from a statsd metric name by a set of
+converter that was configured by user.
+
+```
+converters:
+  - pattern: "cluster.cds_{traffic}_{mesh}_{service}-vn_{protocol}_{port}.{action}"
+    dimensions:
+      - traffic
+      - mesh
+      - service
+      - action
+```
+
+This converter will parse `traffic`, `mesh`, `service` and `action` as dimensions
+from a metric name `cluster.cds_egress_ecommerce-demo-mesh_gateway-vn_tcp_8080.update_success`.
+
+Metrics will be sent without dimensions if there's no related configuration.
+
+#### Formatting metric name
+
+You can customize a metric name by providing a format string within the converter configuration.
+
+```
+converters:
+  - pattern: "cluster.cds_{traffic}_{mesh}_{service}-vn_{protocol}_{port}.{action}"
```

Comment:
Again, this is confusing.  Are those other four sections turned into dimensions or not?  It seems like they should be if they are bothered to be named.